### PR TITLE
Jinja2Bear: Improve documentation

### PR DESCRIPTION
**Pull request by sils**
Related to https://github.com/coala/coala/issues/3488

### CODE REVIEW

File: bears/jinja2/Jinja2Bear.py [hunk 1]
```diff
@@ -312,16 +312,17 @@ def run(self,
...
         * Variable spacing:
             Variable tags should be padded with one space on each side, like
             this: ``{{ var_name }}``. This can be set to any number of spaces
-            via the setting variable_spacing. Malformatted variable tags are
-            detected and fixes suggested.
+            via the setting ``variable_spacing``.
+            Malformatted variable tags are detected and fixes suggested.
         * Control spacing:
-            Like variable spacing, but for control blocks, i.e. if and for
+            Like variable spacing, but for control blocks, i.e., if and for
```
Comment (NiklasMM) [inline bears/jinja2/Jinja2Bear.py]: The comma after the `i.e.` [is apparently only needed in American English](https://jakubmarian.com/comma-after-i-e-and-e-g/). Just throwing this in there as a nice to know, if you don't already. ;)

**Comment by NiklasMM** [comment]
ack  	943fdde

**Comment by sils** [comment]
damn, travis fails for all builds still :/

File: bears/jinja2/Jinja2Bear.py (hunk 1 above)
Comment (Mixih) [inline bears/jinja2/Jinja2Bear.py]: backticks for `if` and `for`?

Comment (sils) [inline bears/jinja2/Jinja2Bear.py]: will do once the build is ready again :/

**Comment by sils** [comment]
0abe3d7 reack

**Comment by sils** [comment]
@rultor merge

**Pull request merged**